.travis.yml

```diff
@@ -65,6 +72,10 @@ before_install:
     sudo sysctl -w kern.sysv.shmseg=128
     export PYTHONPATH=$(pwd)/mason_packages/.link/lib/python/site-packages;
     brew install md5sha1sum;
+    # osrm-backend picks up ccache automatically
+    # since osx does not support caching we want to avoid
+    # since since it will just slow down builds
+    brew rm ccache || true;
```

Comment:
For the record there's also a CMake option `-DENABLE_CCACHE=OFF` since we also had to turn it off from time to time.